plugins/videobasedtracker/BeaconBasedPoseEstimator.cpp

```diff
@@ -447,9 +461,12 @@ namespace vbtracker {
                           out);
         return true;
     }
-
+#if 0
```

Comment:
Leave a comment explaining why this is commented out if you're going to leave it in the codebase.